.codecov.yml

```diff
@@ -0,0 +1,3 @@
+codecov:
```

Comment:
will this append or overwrite main global config? (global config specify coverage policy)

Reply:
other trick is use `_test.go` files, tests don't count into golang coverage